system/Controller.php

```diff
@@ -175,15 +179,37 @@ protected function loadHelpers()
 	 * A shortcut to performing validation on input data. If validation
 	 * is not successful, a $errors property will be set on this class.
 	 *
-	 * @param array  $rules
-	 * @param array  $messages An array of custom error messages
+	 * @param array $rules
+	 * @param array $messages An array of custom error messages
 	 *
-	 * @return bool
+	 * @return boolean
 	 */
 	public function validate($rules, array $messages = []): bool
 	{
 		$this->validator = Services::validation();
 
+		// If you replace the $rules array with the name of the group
+		if (is_string($rules))
+		{
+			$validation = new \Config\Validation();
+
+			// If the rule wasn't found in the \Config\Validation, we
+			// should throw an exception so the developer can find it.
+			if (! isset($validation->$rules))
+			{
+				throw ValidationException::forRuleNotFound($rules);
+			}
+
+			// If no error message is defined, use the error message in the Config\Validation file
+			if (count($messages) === 0)
```

Comment:
use truthy check `if (! $messages)` for better performance: 

```suggestion
			if (! $messages)
```

Reply:
Updated.thank you.